lib/fpm/util.rb

```diff
@@ -226,24 +232,73 @@ def safesystemout(*args)
     return stdout_r_str
   end # def safesystemout
 
+  # Get an array containing the recommended 'ar' command for this platform
+  # and the recommended options to quickly create/append to an archive
+  # without timestamps or uids (if possible).
+  def ar_cmd
```

Comment:
This seems to default to calling `ar` in a way that doesn't use timestamps or uids. This is a behavior change,but perhaps not a problem.

However, if the user is asking for repeatable builds (with whatever flags) on a system who's `ar` is not capable of doing this, then this `ar_cmd` will fall back to one that is not repeatable.

My main goal; If a user is asking for deterministic builds, and the system cannot produce it, we should abort and error. Looking at this `ar_cmd` it is unclear if that is what is happening.